benchmark/common.js

```diff
@@ -3,17 +3,18 @@
 const child_process = require('child_process');
 const http_benchmarkers = require('./_http-benchmarkers.js');
 
-exports.createBenchmark = function(fn, options) {
-  return new Benchmark(fn, options);
+exports.createBenchmark = function(fn, options, flags) {
```

Comment:
I agree. I also noticed that the docs name the second argument `configs`, so I will use `configs` as the local variable name here.